onnxruntime/python/tools/tensorrt/perf/perf.sh

```diff
@@ -5,49 +5,64 @@ FAIL_MODEL_FILE=".fail_model_map"
 LATENCY_FILE=".latency_map"
 METRICS_FILE=".metrics_map"
 
-# symbolic shape info
+# files to download info
 SYMBOLIC_SHAPE_INFER="symbolic_shape_infer.py"
 SYMBOLIC_SHAPE_INFER_LINK="https://raw.githubusercontent.com/microsoft/onnxruntime/master/onnxruntime/python/tools/symbolic_shape_infer.py"
+FLOAT_16="float16.py"
+FLOAT_16_LINK="https://raw.githubusercontent.com/microsoft/onnxconverter-common/master/onnxconverter_common/float16.py"
+
+# root working directory 
+DEFAULT_DIR="/home/olivia/build_cuda/perf/"
```

Comment:
this doesn't seem to be a good default  (should be a generic location and not for specific user)